src/main/resources/db_scripts/createDB.sql

```diff
@@ -0,0 +1,64 @@
+CREATE TABLE question (
```

Comment:
Please do not include this create DB file inside this pull request. It is in other pull request so it is not needed there.